ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/ComplianceLevelsByStandard.tsx

```diff
@@ -129,7 +129,7 @@ function ComplianceLevelsByStandard() {
             header={
                 <Flex direction={{ default: 'row' }}>
                     <FlexItem grow={{ default: 'grow' }}>
-                        <Title headingLevel="h2">Compliance coverage by standard</Title>
+                        <Title headingLevel="h2">Compliance by standard</Title>
```

Comment:
This buys us 70px on the minimum widget size, which translates into 140px of extra space before we drop to a single column layout.